README.md

```diff
@@ -299,7 +299,19 @@ Default: Unset
 
 Valid Values: `stdout` or a file path
 
-Specifies where to write the logging output. Either to stdout or to override the default file.
+Specifies where to write the logging output of L-IPamD. Either to stdout or to override the default file.
```

Comment:
Nit: This is not really just "ipamd", is it? It's the plugin binary as well.

Reply:
You are right, thanks for catching that. I updated the readme.